.github/workflows/_publish-container.yml

```diff
@@ -15,10 +15,7 @@ on:
       version:
         required: true
         type: string
-      docker_context:
-        required: true
-        type: string
-      docker_file:
+      api_location:
```

Comment:
I was a bit confused as location made me think of West EU, East US. Could we call it `project_file`? It's one of those names where it's hard to see it as a newbie.

But I think we will use this also for background jobs, so API might be to specific anyway.